cves/CVE-2016-7401.yml

```diff
@@ -111,10 +107,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: |
+    There were unit tests for cookies originally, but they failed to catch this 
```

Comment:
Explaining what the original unit tests were testing for might help explain to the reader why this bug was not caught.

Reply:
Updated to state that some sanitization checks were missing along with testing for this edge case.